### DISCUSSION TITLE
How to pre-create collections, users, RBAC from helm values?

### DISCUSSION THREAD

**Discussion by sharbuz-truststamp** [milvus-io/milvus#32713] [Q&A and General discussion]
Is it possible to pre-create any users, collections names or RBAC during initial setup using helm chart?

**Comment by yhmo** [comment]
Not possible.

**Comment by xiaofan-luan** [comment]
@LoveEachDay 
any comments?

**Reply by sharbuz-truststamp** [reply]
no, but it would be a great option to have this feature/ability to configure and manage this stuff from helm
